uaaextras/templates/email/body-password.html

```diff
@@ -15,13 +15,34 @@
         <p>
         <ol>
             <li>
-                Verify your email using this link <a href="{{reset.verifyLink}}">{{reset.verifyLink}}</a>, and follow those instructions to generate your temporary password. Save your generated password before following the next step to log in with that password.
+                Verify your email using this link <a href="{{reset.verifyLink}}">{{reset.verifyLink}}</a>
             </li>
             <li>
-                After you log in, you can change your password using this link <a href="{{password.changeLink}}">{{password.changeLink}}</a>.
+                Enter your email address into the reset password verification page
+            </li>
+            <li>
+                Your email is verified and then copy/save the temporary, generated password.
+            </li>
+            <li>
+                <a href="https://login.fr.cloud.gov/">Login to your cloud.gov account</a> with the new temporary password
+            </li>
+            <li>
+                After you log in, you change your temporary password using this link <a href="{{password.changeLink}}">{{password.changeLink}}</a>.
+            </li>
+            <li>
+                Enter the temporary password into the "Old Password" field
+            </li>
+            <li>
+                Then create and enter the new password into "New Password" and confirm it in the "Repeat New Password" fields
             </li>
         </ol>
         </p>
+        <p>
+            More information on accounts is available in our
+            <a href="https://cloud.gov/docs/getting-started/accounts/#log-into-cloudgov">
+                documentation
+            </a>.
+        </p>
         <p>
             If you run into problems or have any questions,
             please email us at
```

Comment:
The email address below this line is incorrect, it should just be `support@cloud.gov` - it might not be a bad idea to change that, hopefully this suggestion works!

```suggestion
            please email us at
            <a href="mailto:support@cloud.gov">support@cloud.gov</a>
```

Reply:
Note that this may not actually change the line below with the email address, GitHub didn't give me an option to comment on that line directly.

Reply:
Ah yes.  Totally missed that. 🎉